examples/unwrap_script.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+""" This script will:
+- download 2-echo phase fieldmap data
+- do the subtraction
+- unwrap phase difference
+- save wrapped and unwrapped plot of first X,Y volume as myplot.png in current directory
+"""
+
+import os
+import numpy as np
+import matplotlib.pyplot as plt
+from zipfile import ZipFile
+import glob
+
+import nibabel as nib
+
+from shimmingtoolbox.unwrap import unwrap_phase
+
+
+def main():
+
+    # Download example data
+    url = 'https://github.com/shimming-toolbox/data-testing/archive/r20200709.zip'
+    filename = 'data-testing.zip'
+    # TODO: replace with download function
+    os.system('curl -o {} -L {}'.format(filename, url))
+    with ZipFile(filename, 'r') as zipObj:
+        # Extract all the contents of zip file in current directory
+        zipObj.extractall()
+    os.remove(filename)
+    # TODO: use systematic name for data-testing (could be in metadata of shimmingtoolbox
+    path_data = glob.glob('data-test*')[0]
+
+    # Open phase data
+    fname_phases = glob.glob(os.path.join(path_data, 'sub-fieldmap', 'fmap', '*phase*.nii.gz'))
+    nii_phase_e1 = nib.load(fname_phases[0])
```

Comment:
When hardcoding indexes in an array, try to validate the length or number to check if it correctly took 2 spaces. If there's one missing, it will raise an error so it should be fine, but if it parses wrongly and creates 3 wrong fname_phases, you might have weird behaviors.

```suggestion
    if len(fname_phases) > 2:
        raise IndexError('Phase data parsing is wrongly parsed or something like this')
    nii_phase_e1 = nib.load(fname_phases[0])
```